.github/workflows/build.yml

```diff
@@ -0,0 +1,82 @@
+name: Build SDK for latest API version
+
+on: 
+  pull_request:
+    branches:
+      - master
+  schedule:
+    # Run on midnight UTC on the first of jan/apr/jul/oct
+    # API is released 1PM EDT/EST
+    - cron: '59 23 1 1/3 *'
+
+jobs:
+  build:
+
+    runs-on: macOS-latest
+
+    steps:
+    - name: Get API Version
+      id: version
+      run: |
+        API_VERSION="$(date +'%Y-%m')"
+        echo "::set-env name=API_VERSION::$API_VERSION"
+
+    - uses: actions/checkout@v2
+    - name: Set up Ruby 2.6
+      uses: actions/setup-ruby@v1
+      with:
+        version: 2.6.x
+
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+        server-id: github # Value of the distributionManagement/repository/id field of the pom.xml
+        settings-path: ${{ github.workspace }} # location for the settings.xml file
+
+    - uses: actions/checkout@master
+      with:
+        ref: 'develop' # This should change to master after 2020-04
+
+    - name: Update git submodules
+      run: git submodule update --init --recursive
+
+    - name: Install bundler
+      run: |
+        gem install bundler
+        BUNDLE_GEMFILE="MobileBuy/buy3/Gemfile" bundle install --jobs 4 --retry 3 --verbose
```

Comment:
Why the jobs and retry?

Reply:
Oops. Since the bundle was failing we were trying various options. I think during that time the `--retry` and `--jobs` options crept in. Will remove that